Added more entries (#187)
nix-community/awesome-nix: README.md

* [nixos-infect](https://github.com/elitak/nixos-infect) - Replace a running non-NixOS Linux host with NixOS.
* [nixos-up](https://github.com/samuela/nixos-up) - Super easy NixOS installer that can be used from the installation ISO.
* [nix-installer](https://github.com/DeterminateSystems/nix-installer) - Opinonated alternative to the official Nix install scripts.

> * [nix-installer-scripts](https://github.com/dnkmmr69420/nix-installer-scripts) - Runs the official installer but does some tweaking as well such as adding fcontext for selinux and installing nix outside of the default profile so you don't accidently uninstall it.

Reviewer: ```suggestion
* [nix-installer-scripts](https://github.com/dnkmmr69420/nix-installer-scripts) - Runs the official Nix installer with some tweaks, such as adding `fcontext` for SELinux and installing Nix outside of the default profile (so you don't accidentally uninstall it).
```
Add a comma after "as well", surround "fcontext" with backticks

* [nixpkgs-firefox-darwin](https://github.com/bandithedoge/nixpkgs-firefox-darwin) - Automatically updated Firefox binary packages for macOS.
* [nixpkgs-wayland](https://github.com/nix-community/nixpkgs-wayland) - Bleeding-edge Wayland packages.
* [NUR](https://github.com/nix-community/NUR/) - Nix User Repositories. The mother of all overlays, allowing access to user repositories and installing packages via attributes.

> * [System Manager](https://github.com/numtide/system-manager) - A system configuration tool using nix for non NixOS distros

Reviewer: ```suggestion
* [System Manager](https://github.com/numtide/system-manager) - A non-NixOS Linux system configuration tool built on Nix.
```

Capitalize "nix", add a hyphen for "non-NixOS" and add ending punctuation too. Consider above rewording as well.
